common/flatpak-dir.c

```diff
@@ -10563,8 +10563,11 @@ flatpak_dir_lookup_cached_summary (FlatpakDir *self,
   if (summary)
     {
       guint64 now = g_get_monotonic_time ();
-      if ((now - summary->time) < (1000 * 1000 * (SUMMARY_CACHE_TIMEOUT_SEC)) &&
-          strcmp (url, summary->url) == 0)
+      if (strcmp (url, summary->url) != 0)
+        {
+          res = FALSE;
```

Comment:
Shouldn't we remove the cached summary if the remote changed url since then too?